internals/search_fulltext.py

```diff
@@ -46,77 +46,91 @@ class FeatureWords(ndb.Model):
+      'web_dev_views_link': [fe.web_dev_views_link],
+      'ff_views_notes': [fe.ff_views_notes],
+      'safari_views_notes': [fe.safari_views_notes],
+      'web_dev_views_notes': [fe.web_dev_views_notes],
+      'other_views_notes': [fe.other_views_notes],
+
+      'security_risks': [fe.security_risks],
+      # TODO: security_review_status
+      # TODO: privacy_review_status
+
+      'ergonomics_risks': [fe.ergonomics_risks],
+      'wpt_descr': [fe.wpt_descr],
+      'webview_risks': [fe.webview_risks],
+
+      'devrel_emails': fe.devrel_emails,
+      'debuggability': [fe.debuggability],
+      'doc_links': fe.doc_links,
+      'sample_links': fe.sample_links,
+      }
+
+
+def get_strings(fe: FeatureEntry, field_name: str|None = None) -> list[str]:
   """Return a list of separate string values in the given feature entry."""
-  strings : list[Optional[str]] = []
-  strings.append(fe.creator_email)
-  strings.append(fe.updater_email)
-  strings.extend(fe.owner_emails)
-  strings.extend(fe.editor_emails)
-  strings.extend(fe.cc_emails)
-
-  strings.append(fe.name)
-  strings.append(fe.summary)
-  # TODO: category
-  strings.extend(fe.blink_components)
-  strings.extend(fe.search_tags)
-  strings.append(fe.feature_notes)
-
-  strings.append(fe.bug_url)
-  strings.append(fe.launch_bug_url)
-
-  # TODO: impl_status_Chrome
-  strings.append(fe.flag_name)
-  strings.append(fe.finch_name)
-  strings.append(fe.non_finch_justification)
-  strings.append(fe.ongoing_constraints)
-
-  strings.append(fe.motivation)
-  strings.append(fe.devtrial_instructions)
-  strings.append(fe.activation_risks)
-  strings.append(fe.measurement)
-
-  strings.append(fe.initial_public_proposal_url)
-  strings.extend(fe.explainer_links)
-  # TODO: standard_maturity
-  strings.append(fe.spec_link)
-  strings.extend(fe.spec_mentor_emails)
-  strings.append(fe.interop_compat_risks)
-  strings.append(fe.all_platforms_descr)
-  strings.append(fe.tag_review)
-  # TODO: tag_review_status
-  strings.append(fe.non_oss_deps)
-  strings.append(fe.anticipated_spec_changes)
-
-  # TODO: ff_views
-  # TODO: safari_views
-  # TODO: web_dev_views
-  strings.append(fe.ff_views_link)
-  strings.append(fe.safari_views_link)
-  strings.append(fe.web_dev_views_link)
-  strings.append(fe.ff_views_notes)
-  strings.append(fe.safari_views_notes)
-  strings.append(fe.web_dev_views_notes)
-  strings.append(fe.other_views_notes)
-
-  strings.append(fe.security_risks)
-  # TODO: security_review_status
-  # TODO: privacy_review_status
-
-  strings.append(fe.ergonomics_risks)
-  strings.append(fe.wpt_descr)
-  strings.append(fe.webview_risks)
-
-  strings.extend(fe.devrel_emails)
-  strings.append(fe.debuggability)
-  strings.extend(fe.doc_links)
-  strings.extend(fe.sample_links)
+  strings_dict = _get_strings_dict(fe)
+
+  if field_name and field_name in strings_dict:
+    strings = strings_dict[field_name]
+  else:
+    strings = [s for s_list in strings_dict.values() for s in s_list]
```

Comment:
Do we have to check None for  s_list? 

Also nit: if it is not more memory efficient to write two for loops in one [] statement, write two separate for loops statement instead

```suggestion
    strings = [s for s_list in strings_dict.values() for s in s_list]
```

Reply:
Actually None is checked below. Ignore my first comment

Reply:
I wrote two because I am iterating over a nested list to flatten the list.  I believe that this is a python idiom for flattening a 2-level list.